docs/components/modeler/web-modeler/process-applications.md

```diff
@@ -36,9 +37,13 @@ The new process application will contain an empty BPMN diagram with the same nam
 :::note
 A process application must always have a main process.
 That is why the main process diagram cannot be deleted or moved out of the process application.
+However, you can assign another BPMN diagram as the new main process with the **Reassign main process** option in the
+diagram's action menu:
 
-To replace the main process' content, upload a local file with the **Replace via upload** option in the breadcrumb menu
-or by dragging and dropping it onto the modeling canvas.
+<p><img src={ReassignMainProcessImg} alt="Reassign the main process" /></p>
+
+Alternatively, replace the main process' content by uploading a local file with the **Replace via upload** option in the
+diagram page's breadcrumb menu or by dragging and dropping it onto the modeling canvas.
```

Comment:
@HanselIdes I kept this sentence for the sake of completeness, but maybe we can also remove it completely now (given the new "Reassign main process" option).

Reply:
Thanks! I'll remove it.